### PR TITLE
Update scheduled-events with information about session object

### PR DESCRIPTION
**Pull request by jveldhuizen**
This change should also be applied to the Mendix 9 documentation.

### CODE REVIEW

**Comment by jveldhuizen** [comment]
Approval needed from Runtime Core.

**Review by hugomendix — APPROVED** [review]
LGTM

PS: If we decide that we want to remedy this in the next major version, we may want to announce that here as well.

File: content/en/docs/refguide/modeling/resources/scheduled-events.md [hunk 1]
```diff
@@ -180,3 +180,7 @@ Hour- and minute-based intervals can only be integer divisors of 24 or 60, respe
 ### 5.4 Cleaning Up Completed Runs of Scheduled Events
 
 Every time a scheduled event is run it produces an entry in the `System.ProcessedQueueTask` table in the database. Over time these accumulate and the table can grow large. Refer to the documentation on [Cleaning Up Old Processed Tasks](/refguide/task-queue/#cleanup) in *Task Queue* to learn how to remove processed entries.
+
+### 5.5 One Session for All Scheduled Events
+
+Each runtime node has one specific session in memory which is used for all scheduled events. Changes to this session are visible for all scheduled events on that node. Things like changing the time zone via a Java action in one scheduled event can lead to unexpected behavior in other scheduled events. You are therefore strongly discouraged from changing the session for scheduled events.
```
Comment (MarkvanMents) [inline content/en/docs/refguide/modeling/resources/scheduled-events.md]: Hi @jveldhuizen 
I'm not sure about the last sentence. It implies that you can change the session, but I don't believe that it is possible to change that. I think we are talking about changing the session _environment_ (timezone, language, time). I'm not sure how best to say this? Perhaps
```suggestion
Each runtime node has one specific session in memory which is used for all scheduled events. Changes to this session are visible for all scheduled events on that node. Things like changing the time zone via a Java action in one scheduled event can lead to unexpected behavior in other scheduled events. You are therefore strongly discouraged from changing the global session environment for scheduled events.
```

@NielsKunst - I will copy this into the Mendix 9 documentation once this language has been agreed.

**Review by MarkvanMents — CHANGES_REQUESTED** [review]
Hi @jveldhuizen 
Jonathan, I have a small suggestion to clarify what we are talking about. I think the current description could be misinterpreted.

File: content/en/docs/refguide/modeling/resources/scheduled-events.md (hunk 1 above)
Comment (jveldhuizen) [reply content/en/docs/refguide/modeling/resources/scheduled-events.md]: You cannot change the session of scheduled events to another session. That's true. But you can change the session object itself; you can change its timezone field, see the method `setTimeZone` on https://apidocs.rnd.mendix.com/10/runtime/com/mendix/systemwideinterfaces/core/ISession.html. If you think that that's more clear with talking about _session environment_, that's fine.

Comment (MarkvanMents) [reply content/en/docs/refguide/modeling/resources/scheduled-events.md]: > You cannot change the session of scheduled events to another session. That's true. But you can change the session object itself; you can change its timezone field, see the method `setTimeZone` on https://apidocs.rnd.mendix.com/10/runtime/com/mendix/systemwideinterfaces/core/ISession.html. If you think that that's more clear with talking about _session environment_, that's fine.

Ah, I see what you mean. How about I say
```suggestion
Each runtime node has one specific session in memory which is used for all scheduled events. Changes to this session are visible for all scheduled events on that node. Things like changing the time zone via a Java action in one scheduled event can lead to unexpected behavior in other scheduled events. You are therefore strongly discouraged from changing the session object for scheduled events.
```

Comment (jveldhuizen) [reply content/en/docs/refguide/modeling/resources/scheduled-events.md]: Yes, that's a nice addition.

**Review by MarkvanMents — APPROVED** [review]
@jveldhuizen 
Looks good, I assume I can merge this immediately? I'll do that unless I hear from you today.

**Pull request merged**